menuinst/_schema.py

```diff
@@ -171,6 +171,11 @@ class Linux(BasePlatformSpecific):
     """
     PrefersNonDefaultGPU: Optional[bool] = None
     "Hint that the app prefers to be run on a more powerful discrete GPU if available."
+    SingleMainWindow: Optional[bool] = None
+    """
+    Hint to avoid offering a UI to launch another window of the app.
+    This is useful for removing the 'New Window' option in the app's context menu.
```

Comment:
```suggestion
    Do not show the 'New Window' option in the app's context menu.
```

This seems more direct. Doesn't have to be adopted exactly like that, but I think the way it's written now is a little difficult to understand.